tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -428,6 +428,10 @@ def _test_cpu_wrmsr_snapshot(context):
     msrs_before_fname = Path(snapshot_artifacts_dir) / shared_names["msrs_before_fname"]
 
     dump_msr_state_to_file(msrs_before_fname, vm.ssh, shared_names)
+    # On T2A, the restore test fails with error "cannot allocate memory" so,
+    # adding delay below as a workaround to unblock the tests for now.
+    # TODO: Debug the issue and remove this delay.
```

Comment:
Looks like we decided some time back that all new TODOs in the code need a reference to a related Github issue.